nba_py/__init__.py

```diff
@@ -1,16 +1,30 @@
 from requests import get
-from datetime import datetime
+from datetime import datetime, timedelta
 from nba_py.constants import League
+import os
 
 HAS_PANDAS = True
 try:
     from pandas import DataFrame
 except ImportError:
     HAS_PANDAS = False
 
+HAS_REQUESTS_CACHE = True
```

Comment:
@mwilliammyers partially yes. But lot of fixes not exist in master and any other branch. From master I only took part which properly creating headers to make api calls up and running - which I believe is necessary to have to library usable. I didn't removed anything from branch code. Well everything is in commits.